python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -1089,3 +1089,37 @@ def test_multiindex_sort_values(pmidx, ascending, return_indexer):
         actual = actual[0]
 
     assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize(
+    "pdi",
+    [
+        pd.MultiIndex(
+            levels=[[1, 3.0, 4, 5], [1, 2.3, 5]],
+            codes=[[0, 0, 1, 2, 3], [0, 2, 1, 1, 0]],
+            names=["x", "y"],
+        ),
+        pd.MultiIndex(
+            levels=[[1, 3, 4, -10], [1, 11, 5]],
+            codes=[[0, 0, 1, 2, 3], [0, 2, 1, 1, 0]],
+            names=["x", "y"],
+        ),
+        pd.MultiIndex(
+            levels=[["a", "b", "c", "100"], ["1", "100", "5"]],
+            codes=[[0, 0, 1, 2, 3], [0, 2, 1, 1, 0]],
+            names=["x", "y"],
+        ),
+    ],
+)
+@pytest.mark.parametrize("ascending", [True, False])
```

Comment:
Are we concerned about testing with nulls? Also, do we do a stable sort?

Reply:
IIRC we currently don't use a stable sort

Reply:
Ok, the tests will pass for this specific data but more generally we might not be able to compare directly with Pandas. Just an observation.

Reply:
Added null test, but will fail for current version of pandas due to: https://github.com/pandas-dev/pandas/issues/35584

Reply:
Thanks @galipremsagar!